src/visualization.jl

```diff
@@ -1,19 +1,22 @@
 using Interpolations;
+using Plots
+using PyPlot
+pyplot()
```

Comment:
```suggestion
```
Please remove these dependencies. They make the compilation of Klimakoffer very slow and the test procedures take forever. PyPlot is never used in the main code, but here. And this code is not tested. Moreover, Plots is already loaded here when available:
https://github.com/noeztekin/Klimakoffer.jl/blob/534094f23458fa957511326e301f19fd63d548c9/src/Klimakoffer.jl#L12-L15